src/platform/web.rs

```diff
@@ -65,8 +65,76 @@ pub trait WindowExtWebSys {
     /// For example, by default using the mouse wheel would cause the page to scroll, enabling this
     /// would prevent that.
     ///
+    /// Specifically, this will call `event.preventDefault()` for the following event types:
+    /// `touchstart`, `wheel`, `contextmenu`, `pointerdown`, `pointermove`, `keyup`, and `keydown`.
+    ///
+    /// For fine-grained control over which events call `event.preventDefault()`, call
+    /// `Window::set_prevent_default(false)` and add an event listener on the canvas for each of
+    /// the above event types that selectively calls `event.preventDefault()`.
```

Comment:
```suggestion
    /// For fine-grained control over which events call `event.preventDefault()`, call
    /// `Window::set_prevent_default(false)` and add event listeners on the canvas that
    /// selectively call `event.preventDefault()`.
```
Users don't really have to address all these events.